app/views/users/edit.html.erb

```diff
@@ -43,7 +43,7 @@
       <nav class="hidden m:block" aria-label="Settings">
         <% Constants::Settings::TAB_LIST.each do |possible_tab| %>
           <a class="crayons-link crayons-link--block <% if @tab == possible_tab.downcase.tr(' ', '-') %>crayons-link--current<% end %>" href="/settings/<%= possible_tab.downcase.tr(" ", "-") %>">
-            <%= inline_svg_tag("twemoji/#{possible_tab.downcase.tr(' ', '-')}", aria: true, class: "crayons-icon crayons-icon--default", title: t("views.settings.tab.#{possible_tab}")) %>
+            <%= crayons_icon_tag("twemoji/#{possible_tab.downcase.tr(' ', '-')}", native: true, title: t("views.settings.tab.#{possible_tab}")) %>
```

Comment:
Here we're using `native: true` which adds the `crayons-icon--default` class to prevent color inheritance.